src/I18n/FrozenDate.php

```diff
@@ -145,10 +145,12 @@ public function __construct($time = 'now', $tz = null)
      * - `from` => another Date object representing the "now" date
      * - `format` => a fall back format if the relative time is longer than the duration specified by end
      * - `accuracy` => Specifies how accurate the date should be described (array)
+     *
```

Comment:
do we really need a leading newline?
afaik markdown needs the trailing one.

Reply:
We'll see.  I'll re-adjust if it doesn't work somewhere.